src/client/app/containers/CompareChartContainer.ts

```diff
@@ -180,13 +203,15 @@ function mapStateToProps(state: State, ownProps: CompareChartContainerProps): an
 			yaxis: {
 				title: unitLabel,
 				showgrid: true,
-				gridcolor: '#ddd'
+				gridcolor: '#ddd',
+				fixedrange: true
```

Comment:
Want to check why added fixedrange?

Reply:
I believe to avoid zooming when dragging across the graph. If this is desired behavior it can be omitted. Let me know.

Reply:
Just noting that we don't want to zoom on compare so this is fine.